crates/bcr-ebill-persistence/src/db/bill.rs

```diff
@@ -47,14 +45,23 @@ impl BillStoreApi for SurrealBillStore {
         ids: &[String],
         identity_node_id: &str,
```

Comment:
How is this identity_node_id materialized when bill operations are performed? Asking this bcs we process public Nostr bill chain messages for all identities independent of the currently active one. Those messages could/will trigger cache updates.